flight/tasks/task_base.py

```diff
@@ -0,0 +1,38 @@
+# Standard Library
+import abc
+import threading
+
+class TaskBase():
+    """A task the the drone can perform.
+
+    Responsible for implementing the core logic of the various actions that a
+    drone can take (ex. Movement, Follow, Heal, Decode). Must implement
+    perform().
+
+    Attributes
+    ----------
+    _drone: dronekit.vehicle
```

Comment:
Inconsistent documentation: you have not documented private attributes anywhere else. You probably should attempt to document all attributes in general, even if it's just an inline comment for a private attribute.

Reply:
I have added documentation for all variables.